classifier_and_summarizer/summarizer/summarizer_output.py

```diff
@@ -46,22 +47,26 @@ class StampPage:
     * stamp_position: The order or index of the stamp in the final
     collected of stamp pages
     '''
+
     def __init__(
             self,
             media_index,
+            sentence_index,
             is_embedded_content,
             overlay_title,
             overlay_text,
             overlay_font_style,
             overlay_font_size,
             stamp_position):
         self.media_index = media_index
+        self.sentence_index = sentence_index
         self.is_embedded_content = is_embedded_content
         self.overlay_title = overlay_title
         self.overlay_text = overlay_text
         self.overlay_font_style = overlay_font_style
         self.overlay_font_size = overlay_font_size
         self.stamp_position = stamp_position
+        self.stamp_descriptor_embedding = []
```

Comment:
Add a comment for this too.

Reply:
Updated accordingly. Thanks for the suggestion!